en/documentation/index.md

```diff
@@ -81,69 +144,108 @@ testing support).
 
 Here is a list of popular tools used by Rubyists:
 
-* Linux and cross-platform tools:
-  * [Aptana Studio][19]
-  * [Atom][atom]
-  * [Emacs][20] with [Ruby mode][21] and [Rsense][22]
-  * [Eric][eric]
-  * [Geany][23]
-  * [gedit][24]
-  * [Kate][kate]
-  * [KDevelop][kdevelop]
-  * [NetBeans][36]
-  * [RubyMine][27]
-  * [SciTe][28]
+* Easy
   * [Sublime Text][37]
-  * [Vim][25] with [vim-ruby][26] plugin and [Rsense][22]
   * [Visual Studio Code][vscode]
+  * [Zed][zed]
+* Adept
+  * [RubyMine][27] (paid)
+* Expert
+  * [Emacs][20] with [Ruby mode][21] or [Enhanced Ruby mode][enh-ruby-mode]
+  * [Vim][25] with [vim-ruby][26] plugin
+  * [NeoVim][neovim]
```

Comment:
I'm not sure if we need or should pair editors with experience levels. For example, I started writing Ruby with Vim, then switched to NeoVim for like 6 years, and switched to VS Code like 2 years ago.

Reply:
I think the idea is from an approachability standpoint instead of an end user skill level standpoint.

Reply:
yeah, I meant this more about learning curve of the editor